src/Strategy/Erlang/ConfigParser.hs

```diff
@@ -57,8 +60,11 @@ instance ToJSON ErlValue where
 alphaNumSeq :: [Char]
 alphaNumSeq = ['0' .. '9'] <> ['A' .. 'Z']
 
-parseConfig :: Parser [ErlValue]
-parseConfig = scn *> parseTuple `endBy1` symbol "."
+atom :: Text -> ErlValue
```

Comment:
[nit] What is an atom?
I ask because here `atom` sounds more like a type and something like `atomize` or `atomFromText` sounds a bit more descriptive.

Reply:
It's working as a smart constructor for an Atom.  In my mind it makes sense to name it like a constructor.  If you feel strongly I can change it to a more function-like name.

Reply:
No that makes sense, I don't feel strongly about this at all.